lib/ansible/modules/net_tools/nmcli.py

```diff
@@ -561,16 +565,19 @@
     HAVE_DBUS = False
 
 NM_CLIENT_IMP_ERR = None
+HAVE_NM_CLIENT = True
 try:
     import gi
-    gi.require_version('NMClient', '1.0')
-    gi.require_version('NetworkManager', '1.0')
-
-    from gi.repository import NetworkManager, NMClient
-    HAVE_NM_CLIENT = True
+    gi.require_version('NM', '1.0')
+    from gi.repository import NM
 except (ImportError, ValueError):
-    NM_CLIENT_IMP_ERR = traceback.format_exc()
-    HAVE_NM_CLIENT = False
+    try:
+        gi.require_version('NMClient', '1.0')
```

Comment:
```suggestion
        import gi
        gi.require_version('NMClient', '1.0')
```

Reply:
This is already being imported on line 570.

Reply:
I think if you import this here, it will solve the CI problem.

Reply:
I do not think this is a good idea.
1) this were to fix a bug this PR is not supposed to fix
2) if this fixes the test, imho the test is broken not the code